docs/catalog/cloud/instal-supported-add-ons-and-extensions.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- compute
+- kubernetes
+- role:cloud-engineer
+- size:medium
+---
+
+# Install supported add-ons and extensions
+
+## Description
+Add-ons and extensions covered by the editor support policy provide additional and supported functionality, while allowing you to benefit from the latest performance improvements and energy optimizations throughout your service lifecycle.
+
+## Solution
+
+Use supported Add-ons and extensions to benefit from newer and more efficient services as they become available.
+
+## SCI Impact
+`SCI = (E * I) + M per R`
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Regarding the SCI equation, using supported add-ons and extensions will impact:
+
+- `E`:  Using the latest functionality and features helps increase energy efficiency.
```

Comment:
I'm not sure this is a true statement. New functionality and features may require *more* energy to operate and support the additional options.